### ISSUE TITLE
K6.0.0 Latest module - Html hyperlink title not showing correctly

### ISSUE THREAD

**Issue by LinelabCZ** [Kunena/Kunena-Forum#9019]
Can you please implement this fix? 
```
//echo ModuleKunenaLatest::shortenLink($this->getTopicLink($topic, $this->params->get('sh_url_link'), null, null, KunenaTemplate::getInstance()->tooltips() . ' topictitle', $this->category, true, false), $this->params->get('titlelength'));
				
echo $this->getTopicLink($topic, $this->params->get('sh_url_link'), null, 'title="'.$this->escape($topic->subject).'"', KunenaTemplate::getInstance()->tooltips() . ' topictitle', $this->category, true, false);
```

**Comment by LinelabCZ** [comment]
Respectively: libraries/kunena/src/View/KunenaView.php
`public function getTopicLink(KunenaTopic $topic, $action = null, $content = null, $title = null, $class = null, KunenaCategory $category = null)`

**Comment by xillibit** [comment]
It's used at some places in latest module like here : https://github.com/Kunena/Kunena-Addons/blob/K6.0/modules/kunenalatest/tmpl/default_row.php#L44, without the links shortened i need to checkhow it looks like

**Comment by LinelabCZ** [comment]
Same problem here:
components/com_kunena/template/aurelia/layouts/topic/row/default.php 

`echo $this->getTopicLink($topic, null, null, null, ...`
4. parametr is null

**Comment by Ruud68** [comment]
maybe solved by #9029 can you verify?

**Comment by LinelabCZ** [comment]
Resolved. Thank you!
